app/code/Magento/Sales/Model/Order/Shipment.php

```diff
@@ -93,6 +101,16 @@ class Shipment extends AbstractModel implements EntityInterface, ShipmentInterfa
      */
     protected $orderRepository;
 
+    /**
+     * @var ShipmentCommentRepositoryInterface
+     */
+    protected $shipmentRepository;
```

Comment:
Consider to use `shippmentCommentRepository` or `commentRepository` as variable name.
Another developer could confuse this with a Shipment Repository.